scripts/postinst.sh

```diff
@@ -6,7 +6,7 @@ set -e
 #
 # TODO: This is only tested on Debian.
 #
-if [ "$1" = "configure" ] && [ -d /run/systemd/system ]; then
+if ( [ "$1" = "configure" ] || [ "$1" = "1" ] ) && [ -d /run/systemd/system ]; then
```

Comment:
Uhh does this work in a POSIX shell? I never know what works and what doesn't in /bin/sh

If you're sure then let's keep it. If not, maybe just do a nested if..